source/includes/gcp/_instances.md

```diff
@@ -370,6 +374,7 @@ Required | &nbsp;
 `cpuCount`<br/>*string* | Updated number of vCPUs of instance.
 `memoryInGB`<br/>*string* | Updated memory of instance.
 `osImageSelfLink`<br/>*string* | The full URL to the OS image.
+`subnetworkName`<br/>*string* | The subnet to which the instance is to be attached upon creation _(must be in the same region as the region provided for the instance)_.
```

Comment:
This is a little wordy. Could we change it to "The subnet that the instance is attached to upon creation. _(Note that the subnet must be in the same region as the instance)_." Or something like that? 
(Note, as long as we avoid saying region twice in the italics and we get rid of "to which to be" I will be happy).

Reply:
Reworded in 037007c39964945a94b30d0906e746d2fcf19f2e